### ISSUE TITLE
Compiler gets rid of "continue" statements in GML

### ISSUE THREAD

**Issue by HaydenAbler** [krzys-h/UndertaleModTool#363]
If your code contains a "continue" statement, then the compiler will get rid of it. This problem arose when editing an existing script in the game, "gml_Script_scr_namingscreen.gml". There are 4 instances of code like this:

```
if ((selected_col < 0) || ((selected_row < 0) || (string_length(charmap[selected_row, selected_col]) > 0)))
{
}
else
    continue
```

And when you re-compile it in the UMT editor, and then view it again, all 4 instances show up like this:
```
if ((selected_col < 0) || ((selected_row < 0) || (string_length(charmap[selected_row, selected_col]) > 0)))
{
}
```

This is a big game-breaking bug, because this scripting change makes it so that if you start the game completely fresh with no save file, then you won't be able to get past the initial "Instruction" screen. If you attempt to select the "Begin Game" option, then the game will freeze, and you'll need to use the task manager to exit out of the game.

**Comment by Grossley** [comment]
This is not a compiler issue, this is a decompiler issue.

**Comment by Grossley** [comment]
The compiler does not, in fact, remove the continue from the disassembly. However, the decompiler isn’t able to identify the continue from the disassembly, so it does not display. The reason it freezes is a more general issue of decompilation errors. Furthermore, this issue is a duplicate of an existing issue, and so will be closed.

**Comment by HaydenAbler** [comment]
Oh, I see, there was originally supposed to be a "break" statement in that if block, and that got removed by the decompiler. While the compiler is still getting rid of that continue statement, it's no surprise that something weird would happen if you write an empty if statement like that, since that's a really weird way of coding things up. I was able to get it to compile correctly by re-inserting a "break" statement into the if block without brackets, and it stayed. I've noticed the UTMT compiler or decompiler doesn't seem to like keeping the brackets with single-line blocks, so maybe that's what was causing problems.
